README.md

```diff
@@ -1,24 +1,37 @@
-# Ukrmap Rails [![Build Status](https://travis-ci.org/ukrmap/ukrmap_rails.svg?branch=master)](https://travis-ci.org/ukrmap/ukrmap_rails)
+# Ukrmap
+  Rube's version 2.3.1
+  Rail's version 5.0.1
```

Comment:
Ruby and Rails version is not necessary here. Please add repo description instead. 
Also please separate it with one empty line.